conan/tools/meson/meson.py

```diff
@@ -84,6 +87,8 @@ def install(self):
             cmd += " " + verbosity
         if self._conanfile.conf.get("tools.build:install_strip", check_type=bool):
             cmd += " --strip"
+        if cli_args:
+            cmd += " " + " ".join(filter(None, cli_args))
```

Comment:
In theory the ``filter()`` wouldn't be necessary, as we don't expect users to provide ``None`` arguments in the list such as ``meson.install(cli_args=["arg1", None, "arg2"])``. And it is generally better to go from simpler cases that fail than other way around, so if some user passes ``None`` it will fail for them and they will realize and remove it from the argument. But we might be silently hiding issues.
```suggestion
            cmd += " " + " ".join(cli_args)
```